### DISCUSSION TITLE
What is the correct syntax for range query in RESR API?

### DISCUSSION THREAD

**Discussion by taishi55** [supabase/supabase#10166] [Questions] [answered]
What is the correct syntax for range query in RESR API?

**Comment by GaryAustin1** [comment, marked as answer]
You can look in PostgREST docs for most of the REST API info.
Here is range, then count is down below that....
https://postgrest.org/en/stable/api.html#limits-and-pagination
![image](https://user-images.githubusercontent.com/54564956/200441433-d79d6205-c35d-476b-8958-96bcec2f757e.png)
